components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -1691,13 +1691,15 @@ export class GitpodServerImpl implements GitpodServerWithTracing, Disposable {
 
         const teamProjects = await this.projectsService.getTeamProjects(teamId);
         teamProjects.forEach(project => {
-            this.deleteProject(ctx, project.id);
-        })
+            /** no awat */ this.deleteProject(ctx, project.id)
+                .catch(err => {/** ignore */});
+        });
 
         const teamMembers = await this.teamDB.findMembersByTeam(teamId);
         teamMembers.forEach(member => {
-            this.removeTeamMember(ctx, teamId, member.userId);
-        })
+            /** no awat */ this.removeTeamMember(ctx, teamId, member.userId)
+            .catch(err => {/** ignore */});
+        });
```

Comment:
Same here, I think we should await:

```ts
        const teamMembers = await this.teamDB.findMembersByTeam(teamId);
        await Promise.all(teamMembers.map(member => this.removeTeamMember(ctx, teamId, member.userId)));
```

Reply:
Thought the same, but was unsure about the context, so left it as is. We can discuss and adjust separately if needed. :+1: